CSSampleApp/src/main/java/com/example/androidsampleapp/network/NetworkAnalysisActivity.kt

```diff
@@ -16,7 +16,7 @@ class NetworkAnalysisActivity : AppCompatActivity() {
     private lateinit var delayAdapter: ArrayAdapter<Delay>
 
     companion object {
-        const val URL = "https://httpstat.us"
+        const val URL = "https://httpstatus-dev.eu-west-1.csq.fr"
```

Comment:
Are we sure we want this? This is kind of making a CS testing service public. I'd say the public httpstat is good enough for the Sample app. wdyt @asalliou ?

Reply:
@asalliou